src/SassBuilder.php

```diff
@@ -18,29 +18,58 @@ class SassBuilder
     private ?SymfonyStyle $output = null;
 
     /**
-     * @param array<string> $sassPaths
+     * @var array<string, bool|string>
+     */
+    private readonly array $sassOptions;
+
+    /**
+     * @see https://sass-lang.com/documentation/cli/dart-sass/
+     */
+    private const DEFAULT_OPTIONS = [
+        'charset' => true,
+        'style' => 'expanded',
+        'source_map' => true,
+        'embed_source_map' => false,
+        'embed_sources' => false,
+    ];
```

Comment:
Hm, do we need this? It seems like we will have to keep this array in sync with default values in the `SymfonycastsSassExtension` and it's easy to mismatch, i.e. change the defaults in one spot only that will cause issues

Reply:
It's a bit more than that, and my code lacks needed comments it seems :)

Those are the default "sass cli" options.

So they can be different from what we do want as default options. 

Or maybe i'm over-thinking all this.

Reply:
Hm, but sass cli defaults are already defaults upstream, so if we will not pass options to the sass cli they will be defaulted. Or am I missing something?

Reply:
The defaults from the bundle depends on the debug (and they probably could/should depend on the env too). I just wanted a place to store the default CLI values, but if you want, i remove it no problem :) 

Do i still keep some "keys" there ... or do we allow any key/value ?

Reply:
Ah, I probably see the reason now, thanks for explaining!